pvdeg/standards.py

```diff
@@ -14,80 +14,181 @@
 
 # from gaps import ProjectPoints
 
-from . import temperature
-from . import spectral
-from . import utilities
-from . import weather
+from pvdeg import temperature, spectral, utilities, weather
 
+def eff_gap_parameters(
```

Comment:
what is this function used for? isn't it just a copy of standoff?

Reply:
Not just a copy of standoff. It sets up data for another function to calculate an effective standoff.

Reply:
okay, I see it now.